### PR TITLE
plans: Fix bug in show variables plan

### PR DESCRIPTION
**Pull request by shenli**
See: https://github.com/pingcap/tidb/issues/540

### CODE REVIEW

**Comment by qiuyesuifeng** [comment]
LGTM

File: plan/plans/show.go [hunk 1]
```diff
@@ -378,10 +378,16 @@ func (s *ShowPlan) fetchShowVariables(ctx context.Context) error {
...
 		var value string
 		if !s.GlobalScope {
-			// Try to get Session Scope variable value
+			// Try to get Session Scope variable value first.
 			sv, ok := sessionVars.Systems[v.Name]
 			if ok {
 				value = sv
+			} else {
+				// If session scope variable is not set, get the global scope value.
+				value, err = ctx.(variable.GlobalSysVarAccessor).GetGlobalSysVar(ctx, v.Name)
```
Comment (siddontang) [inline plan/plans/show.go]: can we use GetGlobalSysVar directly and bind sys var accessor in ctx?

Comment (shenli) [inline plan/plans/show.go]: Yes this is better.
I will do it in the next pr.

**Comment by siddontang** [comment]
ok，LGTM

**Pull request merged**